AzureCalling/Controllers/CallViewController.swift

```diff
@@ -145,7 +147,16 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
     }
 
     private func openParticipantListDrawer() {
-        let participantListDataSource = ParticipantListDataSource()
+        let participantListDataSource = ParticipantListDataSource(participantsFetcher: getParticipantInfoList)
+        participantListDataSource.createParticipantList()
```

Comment:
Could`createParticipantList()` also been moved into ParticipantListDataSource() class?

Reply:
Moved `createParticipantList()` to be invoked in the inits. Also modified the refresh mechanism since we now have the fetcher in the data sources